tests/__fixtures__/validator/projects/project.yaml

```diff
@@ -0,0 +1,42 @@
+processes:
+  - name: web
+    start_command: npm run start
+    is_cronjob: false
```

Comment:
I might be a bit nazi but I think that having `app`, `git` and `is_enable` first makes it more straightforward to read